charts/rstudio-connect/values.yaml

```diff
@@ -234,6 +259,13 @@ config:
     DataDir: /var/lib/rstudio-connect
   Scheduler:
     InitTimeout: 5m
+  Logging:
+    Enabled: true
+    ServiceLog: STDOUT
+    ServiceLogLevel: INFO    # INFO, WARNING or ERROR
```

Comment:
Do you really want to be this verbose by default? Should we instead default to `WARNING`?